dataland-frontend/src/components/forms/CreateEUTaxonomyForFinancials.vue

```diff
@@ -1,137 +1,339 @@
+                                validation="required"
+                                :options="['None...', ...namesOfFilesToUpload]"
+                              />
+                            </div>
+                            <div>
+                              <UploadFormHeader
+                                :name="euTaxonomyKpiNameMappings.page ?? ''"
+                                :explanation="euTaxonomyKpiInfoMappings.page ?? ''"
+                              />
+                              <FormKit
+                                outer-class="w-100"
+                                type="number"
+                                name="page"
+                                placeholder="Page"
+                                validation-label="Page"
+                              />
+                            </div>
+                          </div>
+                        </FormKit>
+                      </div>
+                    </FormKit>
+                  </div>
+                </div>
+
+                <div class="uploadFormSection">
+                  <div class="col-3 p-3 topicLabel">
+                    <h4 id="addKpis" class="anchor title">Add KPIs</h4>
+                    <p>Select at least one services type to add the related section of KPIs.</p>
+                  </div>
+
+                  <!-- Add KPIs -->
+                  <div class="col-9 formFields">
+                    <!-- Data source -->
+                    <div class="form-field">
+                      <UploadFormHeader
+                        data-test="selectKPIsLabel"
+                        :name="euTaxonomyKpiNameMappings.financialServicesTypes ?? ''"
+                        :explanation="euTaxonomyKpiInfoMappings.financialServicesTypes ?? ''"
+                      />
+
+                      <MultiSelect
+                        v-model="selectedKPIs"
+                        :options="kpisModel"
+                        data-test="MultiSelectfinancialServicesTypes"
+                        name="MultiSelectfinancialServicesTypes"
+                        optionLabel="label"
+                        validation-label="Services Types"
+                        validation="required"
+                        placeholder="Select..."
+                        class="mb-3"
+                      />
+                      <ul v-if="selectedKPIs.length">
+                        <li :key="index" v-for="(file, index) of selectedKPIs">{{ file.label }}</li>
+                      </ul>
+
+                      <PrimeButton
+                        @click="confirmeSelectedKPIs"
+                        data-test="addKpisButton"
+                        :label="selectedKPIs.length ? 'UPDATE KPIS' : 'ADD RELATED KPIS'"
+                      />
+                      <FormKit
+                        v-model="computedFinancialServicesTypes"
+                        type="text"
+                        validationLabel="You must choose and confirm"
```

Comment:
Should be named smth like "Choosing a Financials Services Type and adding KPIs for it" so that the error message is
"Choosing a Financials Services Type and adding KPIs for it is required"

Reply:
ok